content/post/kubedb-v2024.1.31/index.md

```diff
@@ -0,0 +1,537 @@
+        --set kubedb-webhook-server.server.securityContext.seccompProfile.type=RuntimeDefault \
+        --set kubedb-ops-manager.operator.securityContext.seccompProfile.type=RuntimeDefault \
+        --set kubedb-autoscaler.operator.securityContext.seccompProfile.type=RuntimeDefault \
+        --set global.featureGates.Druid=true --set global.featureGates.ZooKeeper=true \
+        --set global.featureGates.MySQL=true -set global.featureGates.PostgreSQL=true 
+```
+
+## Kafka
+In this release, we have improved Kafka Health Checker. The health checker now ensures all kafka brokers are connected in a cluster, messages can be concurrently published and consumed via clients.
+Earlier, KubeDB managed Kafka had only two types of termination policy supports - `WipeOut` and `DoNotTerminate`. This release is bringing support for two more - `Halt` and `Delete`, providing the privilege of keeping PVCs and Secrets on cluster deletion.
+
+### Kafka Connect Cluster
+
+This release introduces `ConnectCluster` for **Kafka**, an awesome tool for building reliable and scalable data pipelines. Its pluggable design makes it possible to build powerful pipelines without writing a single line of code. The initial release of Kafka Connect Cluster is bringing support for Provisioning, Custom-Configuration via kubernetes secret and TLS. You can now enable source/sink connector plugins for `GCS`, `S3`, `MongoDB`, `Postgres` and `MySQL`. This release also introduces support for `Connector` CRD. This enables to easily connect source/sink clients to the `ConnectCluster`. 
+
+Let's assume you have a Kafka cluster `kafka-prod`, provisioned using KubeDB is deployed in a namespace called `demo`. You can now provision a `ConnectCluster` using the following yaml.
+
+```yaml
+apiVersion: kafka.kubedb.com/v1alpha1
+kind: ConnectCluster
+metadata:
+  name: connect-cluster
+  namespace: demo
+spec:
+  connectorPlugins:
+  - gcs-0.13.0
+  - mongodb-1.11.0
+  - mysql-2.4.2.final
+  - postgres-2.4.2.final
+  - s3-2.15.0
+  kafkaRef:
+    name: kafka-prod
+    namespace: demo
+  replicas: 2
+  enableSSL: true
+  tls:
+    issuerRef:
+      apiGroup: cert-manager.io
+      kind: Issuer
+      name: connect-ca-issuer
+  terminationPolicy: WipeOut
+  version: 3.6.0
+```
+
+Once ConnectCluster is ready, you can connect any of supported connector via a `Connector` custom resource. In order to Connect a MongoDB database to the `ConnectCluster` apply the following yaml. Provide a kubernetes secret reference in `spec.configSecret` field which should be containing all the necessary connection information to the sink/source client.
+
+```yaml
+apiVersion: kafka.kubedb.com/v1alpha1
+kind: Connector
+metadata:
+  name: mongo-source
+  namespace: demo
+spec:
+  configSecret:
+    name: mongo-source
+  connectClusterRef:
+    name: connect-cluster
+    namespace: demo
+  terminationPolicy: WipeOut
+```
+
+#### supported versions
+`3.3.2`, `3.4.1`, `3.5.1`, `3.6.0`
```

Comment:
I see, some of them are ascending, some are descending.  
Follow only one.  Descending is better here, I think. 
check for all dbs